src/svg/database-outline.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
+<ellipse cx="256" cy="112" rx="176" ry="80" stroke="black" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Full disclosure: I didn't test this, but doesn't `stroke="black"` override setting the `color` on an `ion-icon`? See https://github.com/ionic-team/ionicons/pull/689

Reply:
Fixed in https://github.com/ionic-team/ionicons/pull/1259/commits/5fad275a2ac742dd57e605e53d30375ec2bbbc95

Reply:
Actually I didn't need to do that. Our [build script already handles this](https://github.com/ionic-team/ionicons/blob/d09ddb640497cafb791a8793c35be10d72ab2645/scripts/build.ts#L109-L115). It's a little confusing because the source SVG isn't modified, just the built SVG in `dist`. As a result, this will appear broken in dev, but in production it will be fine.